ultra/baselines/ddpg/ddpg/policy.py

```diff
@@ -0,0 +1,382 @@
+# code iintegrated from
+#  1- https://github.com/udacity/deep-reinforcement-learning
+#  2- https://github.com/sfujim/TD3/blob/master/TD3.py
```

Comment:
Have the licenses been covered for these files, I see they are both MIT which means we need to carry on their copyright notice.

Reply:
Ask Nhat.